src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -2078,6 +2086,7 @@ static PaDeviceInfo *AllocateDeviceListMemory(PaWasapiHostApiRepresentation *paW
 
     if (paWasapi->deviceCount != 0)
     {
+		UINT32 i;
```

Comment:
Bad indent

Reply:
my bad, VS IDE was not configured to use spaces instead of tabs